package.json

```diff
@@ -28,7 +28,8 @@
     "@theforeman/vendor-dev": "^4.14.0",
     "babel-eslint": "^10.0.0",
     "eslint": "^6.8.0",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "@patternfly/react-catalog-view-extension": "^4.8.126"
```

Comment:
Do we have this in foreman-js.
We've already talked about it, but there is just not the PR linked and I believe that if we have it in `foreman-js`, we can remove it from here, right?

Reply:
It should be available https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L28 and seems to work without it. Dropped

Reply:
Although eslint is not happy about it
```
foreman-remote-execution/webpack/react_app/components/RecentJobsCard/RecentJobsCard.js
  16:8  error  '@patternfly/react-catalog-view-extension' should be listed in the project's dependencies. Run 'npm i -S @patternfly/react-catalog-view-extension' to add it  import/no-extraneous-dependencies
  ```